sdk/metrics-testing/src/main/java/io/opentelemetry/sdk/testing/assertj/metrics/AbstractPointDataAssert.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.testing.assertj.metrics;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.sdk.metrics.data.PointData;
+import io.opentelemetry.sdk.testing.assertj.AttributesAssert;
+import io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions;
+import org.assertj.core.api.AbstractAssert;
+import org.assertj.core.api.Assertions;
+
+/** Test assertions for {@link PointData}. */
+public class AbstractPointDataAssert<
+        PointAssertT extends AbstractPointDataAssert<PointAssertT, PointT>,
```

Comment:
this style of type declaration isn't something we've used in the project up until now. I'm not fundamentally opposed to it, but it's definitely more verbose. @anuraaga any thoughts on this? Otherwise, I would think we'd want to go with `<A,P>` or something like that.

Reply:
If you think A/P is more readable, happy to change.   I chose what I did because I kept messing up the letters, otherwise.

Reply:
let's not block this PR on it. It's a simple thing to change in the future.